tests/regression/22-partitioned_arrays/10-array_octagon.c

```diff
@@ -1,5 +1,6 @@
-// PARAM: --set solver td3 --enable ana.int.interval --enable exp.partition-arrays.enabled  --set ana.activated "['base','threadid','threadflag','escape','expRelation','octagon','mallocWrapper']" --set exp.privatization none
+// PARAM: --set solver td3 --enable ana.int.interval --enable exp.partition-arrays.enabled  --set ana.activated "['base','threadid','threadflag','escape','expRelation','apron','mallocWrapper']" --set exp.privatization none
```

Comment:
This and all the other tests that now use apron should be set up like the apron groups are: `SKIP`, but then in CI workflows explicitly test those groups.

Reply:
I moved them to the `regression/36-apron` folder.